app/lib/features/chat/pages/room_page.dart

```diff
@@ -119,13 +121,14 @@ class RoomPage extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
+    final isChatNG = ref.watch(isActiveProvider(LabsFeature.chatNG));
     return OrientationBuilder(
       builder: (context, orientation) => Scaffold(
         resizeToAvoidBottomInset: orientation == Orientation.portrait,
         body: Column(
           children: [
             appBar(context, ref),
-            ChatRoom(roomId: roomId),
+            isChatNG ? ChatNGRoom(roomId: roomId) : ChatRoom(roomId: roomId),
```

Comment:
I am not super convinced. I'd rather we don't interfere in the existing features at all. I'd rather have the check being done in the router where if `isNG` we give a different set routes with a different set of widgets...